R/LearnerRegrForecastAverage.R

```diff
@@ -57,3 +57,6 @@ LearnerRegrForecastAverage = R6::R6Class("LearnerRegrForecastAverage",
     }
   )
 )
+
```

Comment:
Not your mistake, but I think it might be better to have this consistent with mlr3, i.e. have a "featurelesss" learner that allows to use the median or mean. Does not have to be in this PR though. 

https://github.com/mlr-org/mlr3/blob/main/R/LearnerClassifFeatureless.R